src/ResourceManager/Compute/Commands.Compute/Commands.Compute.csproj

```diff
@@ -48,6 +48,9 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Web.Administration, Version=7.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35">
```

Comment:
This isn't safe.  Microsoft.Web.Administration is platform-specific.  The actual methods in this library are unused in the deployment code.  Closing this PR for now (thanks for looking at this, though)